Source/AudibleUtilities/Widevine/Device.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.IO;
+using System.Security.Cryptography;
+
+#nullable enable
+namespace AudibleUtilities.Widevine;
+
+internal enum DeviceTypes : byte
+{
+	Unknown = 0,
+	Chrome = 1,
+	Android = 2
+}
+
+internal class Device
+{
+	public DeviceTypes Type { get; }
+	public int FileVersion { get; }
+	public int SecurityLevel { get; }
+	public int Flags { get; }
+
+	public RSA CdmKey { get; }
+	internal ClientIdentification ClientId { get; }
+
+	public Device(Span<byte> fileData)
```

Comment:
This is a CDM Devide file compatible with the https://github.com/devine-dl/pywidevine project.